custom_components/daikin_onecta/__init__.py

```
@@ -40,7 +42,9 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
 
     try:
         await daikin_api.async_get_access_token()
-    except ClientError as err:
+    except OAuth2TokenRequestReauthError as err:
+        raise ConfigEntryAuthFailed from err
+    except (OAuth2TokenRequestError, aiohttp.ClientError) as err:
```

Ruff (0.15.20)

[error] Undefined name `aiohttp`

(F821)